webapp/src/js/actions/global.js

```diff
@@ -151,10 +159,40 @@ angular.module('inboxServices').factory('GlobalActions',
         });
       }
 
+      /**
+       * Unset the selected item without navigation
+       */
+      function unsetSelected() {
```

Comment:
This function seems pretty scattergunny, and presumably means that if we added a new page / view that had some kind of selection we'd need to add it here.

I'm not that familiar with redux so I'm not sure if there is a less hardcoded way of doing this, but I wanted to mention it all the same.

Reply:
Yeah I think it would be cleaner to have one unset selected for contacts and one for reports which both call the common code, but I'm comfortable with this for now. In some ways it's quite nice to ensure everything's clean.

I think this will get a lot cleaner when we drop LiveList and just store the selected object in the redux state.